client-v2/src/components/FrontsEdit/FrontContainer.tsx

```diff
@@ -132,13 +152,58 @@ class Fronts extends React.Component<FrontsComponentProps, ComponentState> {
     this.props.frontsActions.editorCloseFront(this.props.frontId);
   };
 
+  public renameFront = () => {
```

Comment:
There are lots of `public` methods on this class -- if we're not intending parent components to use them via refs or whatnot, they might be better as `private`.